docs/change-logs/2022-05-04.md

```diff
@@ -0,0 +1,54 @@
+# 2022-05-04
+
+Current rev: a79e72f69701695185f2f71788d17998bdd5a5a8.
+
+Based on https://github.com/ethereum/go-ethereum v1.10.13.
+
+**Notable changes:**
```

Comment:
Do we include all commits in this change log? If so, maybe just call it "Changes:"

Another suggestion. Let just keep the file in the root folder named with `CHANGELOGS.md` and require every new PRs to update the change log file

Reply:
> Do we include all commits in this change log? If so, maybe just call it "Changes:"

Not every change.

for example, I skip some readme updates & comment updates & formatting.


> Let just keep the file in the root folder named with CHANGELOGS.md and require every new PRs to update the change log file.

My concern is that a single `CHANGELOGS.md` will grow really bigger. But if still think a single file is good I would do it.

Reply:
I‘ve decided to use a single file for now. If it's too big in the future, we then split it.

Reply:
Hopefully the change list won't be too big